### DISCUSSION TITLE
这下褒姒了

### DISCUSSION THREAD

**Discussion by mewoll** [share121/inter-knot#1823] [其他]
绳匠们的配置怎么样，我只能玩1080p 60帧 ，哭了
![Snipaste_2024-08-13_11-04-32](https://github.com/user-attachments/assets/612b563f-6a8c-45bb-9cd2-9a467a4c0df5)
![B1 Screenshot 2024 08 13 - 11 09 54 75](https://github.com/user-attachments/assets/1f74a87a-09f1-40c5-9455-4eefaf3a0747)
![B1 Screenshot 2024 08 13 - 11 12 01 21](https://github.com/user-attachments/assets/08e07638-17af-426f-82ed-c6933afb6091)
![B1 Screenshot 2024 08 13 - 11 12 18 41](https://github.com/user-attachments/assets/c258bb26-4945-44df-b7f6-11c8e5e11eba)

**Comment by kuyuma** [comment]
希望4080 能稳定4k60帧就行

**Comment by molehzy** [comment]
![2a3cb5b61e9c61f58f0268920fe59b22](https://github.com/user-attachments/assets/d5fe6c66-3d41-408f-8b2f-3b81a98bd225)
无脑拉最高

**Comment by Clyinton** [comment]
![1](https://github.com/user-attachments/assets/902ffd8a-ebdb-4aa1-b23c-a51a5e127938)
玩辣！（虽然我没买）

**Comment by SednaMi** [comment]
3060laptop，2k，全高，50差不多75左右

**Comment by Kamisato100** [comment]
57x3d+6600（=3060），默认设置全高80多帧，把采样率拉满，不用FSR和帧生成只有30多帧，优化表现只能说马马虎虎吧
